From dd66452d11aab14359f46eaa53d0af98c6a4d612 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey.doherty@intel.com>
Date: Wed, 16 Jan 2019 12:07:45 +0000
Subject: [PATCH] schemas: Allow GNOME sound to be pushed beyond 100% volume

This is so our users retain full control and are free to blast out
the sounds as merry as can be. OK so it's basically for me to listen
to Metallica but you get the gist.

Signed-off-by: Ikey Doherty <ikey.doherty@intel.com>
---
 schemas/meson.build                              | 1 +
 schemas/org.gnome.desktop.sound.gschema.override | 2 ++
 2 files changed, 3 insertions(+)
 create mode 100644 schemas/org.gnome.desktop.sound.gschema.override

diff --git a/schemas/meson.build b/schemas/meson.build
index 353aebd..4f7728f 100644
--- a/schemas/meson.build
+++ b/schemas/meson.build
@@ -10,6 +10,7 @@ gnome_schemas = [
     'org.gnome.nautilus.preferences.gschema.override',
     'org.gnome.settings-daemon.plugins.power.gschema.override',
     'org.gnome.settings-daemon.plugins.xsettings.gschema.override',
+    'org.gnome.desktop.sound.gschema.override',
     'org.gnome.shell.extensions.dash-to-dock.gschema.override',
     'org.gnome.shell.extensions.desktop-icons.gschema.override',
     'org.gnome.shell.extensions.net.gfxmonk.impatience.gschema.override',
diff --git a/schemas/org.gnome.desktop.sound.gschema.override b/schemas/org.gnome.desktop.sound.gschema.override
new file mode 100644
index 0000000..3bee35a
--- /dev/null
+++ b/schemas/org.gnome.desktop.sound.gschema.override
@@ -0,0 +1,2 @@
+[org.gnome.desktop.sound]
+allow-volume-above-100-percent=true
-- 
2.20.0

